src/style.css

```diff
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: "Avenir";
+  src: local("Avenir"), url(fonts/Avenir.ttc);
+  font-weight: 300px;
+}
+
+form {
+  box-sizing: border-box;
+}
+
+.search-form-big {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.input-big {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+
+  display: inline-block;
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input-big:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input-big:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.input-small {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+
+  display: inline-block;
+  box-sizing: border-box;
+  height: 42px;
+  width: 100%;
+  padding-left: 33px;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input-small:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input-small:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.img1 {
+  position: absolute;
+  display: inline-block;
+  padding: 0;
+  height: 19px;
+  width: 19px;
+  margin-top: 25px;
+  margin-left: 26px;
+}
+
+.img2 {
+  position: absolute;
+  display: inline-block;
+  padding: 0;
+  height: 11px;
+  width: 11px;
+  margin-top: 15px;
+  margin-left: 13px;
```

Comment:
DRY - do not repeat yourself
part of the styles between big and small searches are the same
[checklist ](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md#:~:text=%5BSTYLES%5D%20%2D%20if%20you,fledged%20styled%20element.)